Modules/Quest/QuestieQuest.lua

```diff
@@ -362,9 +362,10 @@ function QuestieQuest:AcceptQuest(questId)
 
 end
 
-function QuestieQuest:CompleteQuest(questId)
+function QuestieQuest:CompleteQuest(quest)
+    local questId = quest.Id
     QuestiePlayer.currentQuestlog[questId] = nil;
-    Questie.db.char.complete[questId] = true --can we use some other relevant info here?
+    Questie.db.char.complete[questId] = quest and not quest.Repeatable
```

Comment:
We don't need to check for `quest ~= nil`, since it is already used before. Instead we just need to check for the `Repeatable` property:

```suggestion
    Questie.db.char.complete[questId] = not quest.Repeatable
```

Reply:
Maybe you could add a small comment to that line as well to explain it for later viewers :ok_hand: 

Also this needs to be tested with regular repeatable quests and with repeatable quests in a child/parent releation (meaning sub quests).